cmd/commands/cli/command_identities.go

```diff
@@ -107,6 +107,8 @@ func (c *cliApp) getIdentity(actionArgs []string) {
 	info("Channel address:", identityStatus.ChannelAddress)
 	info("Balance:", identityStatus.Balance)
 	info("Balance estimate:", identityStatus.BalanceEstimate)
+	info("Earnings:", identityStatus.Earnings)
```

Comment:
end users will be confused by the earnings and earnings total.

Reply:
I would suggest a better name but can't think of any.

Reply:
Source https://github.com/mysteriumnetwork/node/issues/1731.
ATM can't think off anything better :/